expr/applier.go

```diff
@@ -0,0 +1,11 @@
+package expr
+
+import (
+	"fmt"
+)
+
+type Applier interface {
```

Comment:
I think this should stay in proc/apply.go since that's the only place it's used.

Reply:
as we discussed

Reply:
When we get rid of warnings, we won't need the interface anymore anyway!  It will just be expr.Eval.